leetcode/src/75.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+void sortColors(int* arr, int n){
+    void swap(int *x, int *y){
+    if (x==y)
+        return;
+   *x = *x + *y;
+    *y= *x - *y;
+    *x= *x - *y;
+}
+
+    int start=0, mid=0, end=n-1;
+    while(mid<=end){
+        if(arr[mid]==1)
+            mid++;
+        else if(arr[mid]==0){
+            swap(&arr[mid],&arr[start]);
+            mid++;
+            start++;
+        }
+        else{
+            swap(&arr[mid],&arr[end]);
+            end--;
+        }
+    }
+}
+int main(){
```

Comment:
Also, I was under the impression that leetcode problems solutions in the repository shouldn't have the `main` function defined inside them...

Reply:
I've removed the `main` function and made `swap` global.